.scripts/linux/compare-benchmarks.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+set -x
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+REPO_DIR=$SCRIPT_DIR/../../
+cd $REPO_DIR
+
+benchmark/tools/compare.py benchmarks "$1" "$2"
```

Comment:
Looks like this file is missing from the pr?

Reply:
compare.py? it's part of the google benchmarks repo which is checked out

Reply:
Oh I see